examples/support/mod.rs

```diff
@@ -1,6 +1,6 @@
 #![allow(dead_code)]
 
-extern crate obj;
+
```

Comment:
Why do you replace the `extern crate` statements with empty lines? If you remove them, please also remove the line.